src/cmd/ksh93/tests/ifs.sh

```diff
@@ -0,0 +1,30 @@
+# These are the tests for the internal field separator (IFS).
+
+IFS=e
+set : :
+[[ "$*" == ":e:" ]] || log_error "IFS failed" ":e:" "$*"
+
+IFS='|' read -r first second third <<< 'one|two|three'
+[[ "${first}" == "one" ]] || log_error "IFS failed" "one" "${first}"
+[[ "${second}" == "two" ]] || log_error "IFS failed" "two" "${second}"
+[[ "${third}" == "three" ]] || log_error "IFS failed" "three" "${third}"
+
+# Multi-byte character checks will only work if UTF-8 inputs are enabled
+if [ "${LANG}" = "C.UTF-8" ]
```

Comment:
Please use `en_US.UTF-8` rather than `C.UTF-8`. I'm working on killing the AST locale subsystem which means `C.UTF-8` will no longer be supported unless the platform supports it.